src/ClusterManager/job_manager.py

```diff
@@ -169,6 +169,8 @@ def SubmitRegularJob(job):
             jobParams["mountpoints"].append(mp)
 
         userAlias = getAlias(jobParams["userName"])
+        userEmail = userAlias + "@microsoft.com" # assume users are corp users
```

Comment:
I think jobParams["userName"] contains full emailId. If that's the case, then we can use it directly instead of hardcoding microsoft.com.

Reply:
Ok, I'll use that then.